File: order_backend_node/app/model/goods.js
'use strict';

module.exports = app => {
  const { STRING, INTEGER, DECIMAL} = app.Sequelize;
  return app.model.define('goods', {
    id: { type: INTEGER, primaryKey: true, autoIncrement: true },
    name: STRING,
    description: STRING,
    icon: STRING,
    stock: INTEGER,
    price: DECIMAL,
    weight: INTEGER,
    category_id: INTEGER,
    status: INTEGER,
    del: INTEGER,
  });
};

File: order_backend_springboot/src/main/java/com/order/domain/entity/Goods.java
package com.order.domain.entity;

import com.order.enums.GoodsStatusEnum;
import lombok.Data;
import org.hibernate.annotations.DynamicUpdate;

import javax.persistence.*;
import java.math.BigDecimal;
import java.util.Date;

@Entity
@Data
@DynamicUpdate
public class Goods {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private Integer categoryId;

    private String name;

    private String description;

    private String icon = "http://static.c2wei.cn/logo.png";

    private Integer stock = 99;

    private BigDecimal price;

    private Integer weight = 0;

    private Integer status = GoodsStatusEnum.UP.getStatus();

    private Date createTime;

    private Date updateTime;

    private Integer del = 0;
    
    public Goods() {
    }
}
